common:
    lid_topic: "/os_cloud_node/points"
    imu_topic: "/os_cloud_node/imu"
    time_sync_en: false # ONLY turn on when external time synchronization is really not possible

preprocess:
    scan_line: 128
    blind: 4
    metadata_json: "config/metadata_RILIO.json"
    calibration_json: "config/lidar_calibration_RILIO.json"

mapping:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    fov_degree: 180
    det_range: 150.0
    extrinsic_est_en: false # true: enable the online estimation of IMU-LiDAR extrinsic
    extrinsic_T: [-0.006253, 0.011775, -0.07645]
    extrinsic_R: [1, 0, 0, 0, 1, 0, 0, 0, 1]

publish:
    path_en: true
    scan_publish_en: true # false: close all the point cloud output
    dense_publish_en: true # false: low down the points number in a global-frame point clouds scan.
    scan_bodyframe_pub_en: true # true: output the point cloud scans in IMU-body-frame
    show_ref_img: true

pcd_save:
    pcd_save_en: false
    interval:
        -1 # how many LiDAR frames saved in each pcd file;
        # -1 : all frames will be saved in ONE pcd file, may lead to memory crash when having too much frames.
